lib/services/project-service.ts

```diff
@@ -73,11 +73,10 @@ export class ProjectService implements IProjectService {
 	public isValidNativeScriptProject(pathToProject?: string): boolean {
```

Comment:
I was wondering, do we want to return some errors if we can identify. I mean - OK, it's not a valid NativeScript project but what's the actual issue? Can the user repair his project to be valid - in case he can, I'd suggest we somehow give him a hint what needs to be fixed? What do you think?